src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -827,4 +827,10 @@
     <value>Reset tab color</value>
     <comment>Text used to identify the reset button</comment>
   </data>
+    <data name="MoveTabtoNewWindowText" xml:space="preserve">

```

Comment:
```suggestion
    <data name="MoveTabToNewWindowText" xml:space="preserve">
```

suggestion: We've got a handy spellbot that tries to look for typos. It's smart enough to understand PascalCase and camelCase (et. al), so you've got to capitalize the "to" here, so it doesn't think the work is "Tabto"

(same idea for a couple other places)